### PR TITLE
Improve build_download_helper

### PR DESCRIPTION
**Pull request by Felixoid**
### Changelog category (leave one):
- CI Fix or Improvement (changelog entry is not required)

### CODE REVIEW

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit 6b43d2e1971ac9303ca30692740c1469ddd39504 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/61592/6b43d2e1971ac9303ca30692740c1469ddd39504/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>PR Check</td><td>There's no description for the check yet, please add it to tests/ci/ci_config.py:CHECK_DESCRIPTIONS</td><td>✅ success</td></tr>
<tbody>
</table>
</details>

File: tests/ci/build_download_helper.py [hunk 1]
```diff
@@ -183,47 +185,75 @@ def download_build_with_progress(url: str, path: Path) -> None:
...
+def download_clickhouse_binary(
+    check_name: str, reports_path: Union[Path, str], result_path: Path
+) -> None:
     download_builds_filter(
         check_name, reports_path, result_path, lambda x: x.endswith("clickhouse")
     )
 
 
-def download_performance_build(check_name, reports_path, result_path):
+def get_clickhouse_binary_url(
```
Comment (alesapin) [inline tests/ci/build_download_helper.py:237]: What is it?)

Comment (Felixoid) [reply tests/ci/build_download_helper.py:237]: Reducing diff with cloud CI

**Comment by Felixoid** [comment]
https://github.com/ClickHouse/ClickHouse/pull/61604 the RISCV build

**Pull request merged**